client/src/areas/party/home/ItemScoreInfoDialog.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from "react";
+import styled from "styled-components";
+import { Typography } from "@material-ui/core";
+import { FC } from "react";
+import Dialog from "@material-ui/core/Dialog/Dialog";
+import DialogContent from "@material-ui/core/DialogContent/DialogContent";
+import Icon, { IconProps } from "@material-ui/core/Icon/Icon";
+import Table, { TableProps } from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell, { TableCellProps } from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+
+export interface IInfoProps {
+  onDialogClosed: () => any;
+}
+
+export const ItemScoreInfoDialog: FC<IInfoProps> = ({ onDialogClosed }) => {
+  return (
+    <Dialog open={true} onClose={onDialogClosed}>
+      <DialogContent>
+        <div>
+          <CloseIcon onClick={onDialogClosed}>close</CloseIcon>
+          <TitleText variant="h4">
+            Introducing ItemScore! <br />
+            <br />
+          </TitleText>
+          <Typography variant="body1">
+            A unique or set item's "Item Score" is based on it's relative rarity
+            to that ultimate Holy Grail in Diablo II: Tyrael's Might. <br />
```

Comment:
this sentence sound strange...however you're the native speaker... so you know best if it's correct :D

Reply:
It was strange, I edited this on latest commit. Hopefully clearer?